tests/cli/test_flow_run.py

```diff
@@ -192,6 +192,49 @@ def test_ls_state_name_filter(
     )
 
 
+def test_ls_state_name_case_insensitive_filter(
+    scheduled_flow_run,
+    completed_flow_run,
+    running_flow_run,
+    late_flow_run,
+):
+    result = invoke_and_assert(
+        command=["flow-run", "ls", "--state", "late"],
```

Comment:
We could probably just parametrize the existing `test_ls_state_name_filter` with the different `state` arguments (`"late", "Late", etc.) but that doesn't block this PR.